README.md

```
@@ -35,27 +47,49 @@ cd portless-home
 
 ```sh
 ./uninstall.sh
 ```
 
-Removes the LaunchAgent, the installed files, and the `:443` serve rule.
+Removes the login service, the installed files, and the `:443` serve
+rule.
 
 ## How it fits together
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)